module/product/src/Domain/Query/ProductQueryInterface.php

```diff
@@ -117,4 +117,9 @@ public function findAttributeIdsBySku(Sku $sku): array;
      * @return AttributeId[]
      */
     public function findAttributeIdsByProductId(ProductId $productId): array;
+
+    /**
+     * @return array
+     */
+    public function findProductIdsWithBoundAttributeByAttributeId(AggregateId $id): array;
```

Comment:
If an external module decorates this interface, such a change will cause it to generate an error